frontback/frontBackCSHORE.py

```diff
@@ -1040,832 +924,177 @@ def CSHOREsimSetup(startTime, inputDict):
-                assert bc_coords['xFRF'] <= max(master_bathy['xFRF']) and bc_coords['xFRF'] >= min(master_bathy['xFRF']), 'The wave gage selected as the boundary condition is outside the known bathymetry.'
-
-                # make the shift from "master bathy" to model bathy convention (zero at forcing instrument and positive increasing towards shore)
-                BC_dict['x'] = np.flipud(int(round(bc_coords['xFRF'])) - master_bathy['xFRF'][np.argwhere(master_bathy['xFRF'] <= int(round(bc_coords['xFRF'])))].flatten())
-                BC_dict['zb'] = np.flipud(master_bathy['elev'][np.argwhere(master_bathy['xFRF'] <= int(round(bc_coords['xFRF'])))].flatten())
-                BC_dict['fw'] = np.flipud(fric_fac * np.ones(BC_dict['x'].size))  # cross-shore values of the bottom friction (just sets it to be fric_fac at every point in the array)
+        else:
+            # no i am not
+            b_dict = {}
+            try:
+                Time_O = (DT.datetime.strptime(startTime, '%Y-%m-%dT%H:%M:%SZ') - DT.timedelta(days=1)).strftime('%Y-%m-%dT%H%M%SZ')
+                # initialize the class
+                cshore_io_O = inputOutput.cshoreIO()
+                # get into the directory I need
+                path_prefix_O = path_prefix
+                params0, bc0, veg0, hydro0, sed0, morpho0, meta0 = cshore_io_O.load_CSHORE_results(path_prefix_O + Time_O)
+                prev_wg = meta0['BC_gage']
+
+                # which gage was it?
+                frf_Data = getObs(start_time - DT.timedelta(days=0, hours=3, minutes=0), end_time + DT.timedelta(days=0, hours=3, minutes=1), server)
+                wave_data8m = frf_Data.getWaveSpec(gaugenumber=12)
+                wave_data6m = frf_Data.getWaveSpec(gaugenumber=4)
+                t_class = PDL.PrepDataTools()
+                o_dict = t_class.waveTree_CSHORE(wave_data8m, wave_data6m, BC_dict['timebc_wave'], start_time, prev_wg=prev_wg)
+                assert o_dict is not None, 'Simulation broken.  Missing wave data!'
+
+                # bathy stuff here
+                # ____________ BATHY ______________________
+                print('\n____________________\nGetting Bathymetric Data\n')
+                bathy_data = {}
+                bathy_data['meta0'] = meta0
+                bathy_data['morpho0'] = morpho0
+                t_class = PDL.PrepDataTools()
+                b_dict = t_class.prep_CSHOREbathy(bathy_data, 'prior_model', profile_num, dx, o_dict, fric_fac)
+
+                # check to see if we stepped down and I have to adjust my x, zb
+                if prev_wg == wave_data8m['name'] and o_dict['BC_gage'] == wave_data6m['name']:
+                    t_class = PDL.PrepDataTools()
+                    b_dict = t_class.prep_CSHOREbathy(b_dict, o_dict)
+            except:
+                assert 'zb' in list(b_dict.keys()), 'Please begin simulations the day after a new integrated bathymetry'
```

Comment:
please revise to "Please begin simulations the day after a newly collected bathymetry"

Reply:
Fixed.